manifests/v/VSCodium/VSCodium/1.71.0.22245/VSCodium.VSCodium.locale.en-US.yaml

```diff
@@ -1,23 +1,22 @@
-# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
-
 PackageIdentifier: VSCodium.VSCodium
 PackageVersion: 1.71.0.22245
 PackageLocale: en-US
-Publisher: Microsoft Corporation
+Publisher: VSCodium Team
```

Comment:
```suggestion
Publisher: Microsoft Corporation
```
This actually is correct as `Microsoft Corporation` because that is the value which is written to Control Panel when the application is installed
![image](https://user-images.githubusercontent.com/12611259/189117461-9766351c-f9cc-4f9f-b6e2-5b5a8eaee019.png)

Reply:
Run `appwiz.cpl`. In the list that appears, right click on a column name and ensure "Publisher" is checked

Reply:
I've found the missing replacements but only the next release will have the changes...
Should I close this PR and push it on the next release?

Reply:
> Should I close this PR and push it on the next release?

Yes.